src/runtime/rpc/rpc_session.h

```diff
@@ -36,8 +36,29 @@
 namespace tvm {
 namespace runtime {
 
+// Magic header for RPC data plane
 const int kRPCMagic = 0xff271;
+// magic header for RPC tracker(control plane)
+const int kRPCTrackerMagic = 0x2f271;
+// sucess response
+const int kRPCSuccess = kRPCMagic + 0;
+// duplicate key in proxy
+const int kRPCDupicate = kRPCMagic + 1;
```

Comment:
Should this response code be handled somewhere?

Reply:
It is used for proxy mode. However, this PR doesn't implement proxy in fact, so I remove related unnecessary code, currently work like our java standalone server.